File: c/udemy/5-estruturas-repetitivas/loops-definidos/dentro_fora.c
// (adaptado de URI 1072) Leia um valor inteiro N. Este valor será a quantidade de valores inteiros X que serão lidos em seguida. Mostre quantos destes valores X estão dentro do intervalo [10,20] e quantos estão fora do intervalo

#include <stdio.h>

int main() {

    int n, x, dentro = 0, fora = 0;

    printf("Quantos valores a serem lidos? ");
    scanf("%d", &n);

    for (int i = 0; i < n; i++){

        printf("Valor: ");
        scanf("%d", &x);

        if (x >= 10 && x <= 20){
            dentro += 1;
        } else{
            fora += 1;
        }
    }

    printf("\n%d DENTRO", dentro);
    printf("\n%d FORA", fora);

    return 0;

}

File: python/udemy/5-estrutura-repetitiva/loops_definidos/dentro_fora.py
# (adaptado de URI 1072) Leia um valor inteiro N. Este valor será a quantidade de valores inteiros X que serão lidos em seguida. Mostre quantos destes valores X estão dentro do intervalo [10,20] e quantos estão fora do intervalo, conforme exemplo. 

n = int(input('Quantos números você vai digitar? '))
dentro = 0
fora = 0

for i in range(n):
    x = int(input(f'Valor inteiro n. {i+1}: '))
    if x in range(10, 21):
        dentro += 1
    else:
        fora += 1

print(f'{dentro} DENTRO')
print(f'{fora} FORA')